src/atoms/IndexAtom.jl

```diff
@@ -109,11 +109,17 @@ function Base.getindex(x::AbstractExpr, rows::AbstractVector{<:Real}, col::Real)
     return getindex(x, rows, col:col)
 end
 
-function Base.getindex(x::AbstractExpr, I::AbstractMatrix{Bool})
+function Base.getindex(
+    x::AbstractExpr,
+    I::Union{AbstractMatrix{Bool},<:BitMatrix},
+)
     return [xi for (xi, ii) in zip(x, I) if ii]
```

Comment:
Hmm. So these methods have been there forever:
https://github.com/jump-dev/Convex.jl/commit/92c93fd543940cbd3c371a2c1e4367fe23a433d7
but they don't really with the vibe of Convex because they return vector/matrices instead of a new atom.